code/modules/reagents/reagent_containers/chem_bag.dm

```diff
@@ -0,0 +1,70 @@
+/obj/item/reagent_containers/chem_bag
+	name = "chemical bag"
+	desc = "Contains chemicals used for transfusion. Must be attached to an IV drip."
+	icon = 'icons/obj/bloodpack.dmi'
+	icon_state = "bloodpack"
+	volume = 200
+	fill_icon_thresholds = list(10, 20, 30, 40, 50, 60, 70, 80, 90, 100)
+	reagent_flags = TRANSPARENT | ABSOLUTELY_GRINDABLE
+	var/label_name // this is to support when you don't want to display "chemical bag" part with a custom name
```

Comment:
I am thinking to change this standised as all chem types, so Bacon's PR can conveniently use this label, or IV drip can refer this.